front/lib/resources/space_resource.ts

```diff
@@ -517,6 +517,67 @@ export class SpaceResource extends BaseResource<SpaceModel> {
     });
   }
 
+  async manageMember(
+    auth: Authenticator,
+    {
+      userId,
+      operation,
+    }: {
+      userId: string;
+      operation: "add" | "remove";
+    }
+  ): Promise<Result<UserResource, DustError>> {
+    if (!this.canAdministrate(auth)) {
+      return new Err(
+        new DustError(
+          "unauthorized",
+          "You do not have permission to add members to this space."
+        )
+      );
+    }
+
+    const regularGroups = this.groups.filter(
+      (group) => group.kind === "regular"
+    );
+
+    assert(
+      regularGroups.length === 1,
+      `Expected exactly one regular group for the space, but found ${regularGroups.length}.`
+    );
+
+    const [defaultSpaceGroup] = regularGroups;
```

Comment:
It could be easier to have a method that retuen the defaultSpaceGroup (as resource), then use the `addMember(s)` / `removeMember(s)` from here